src/main/java/gov/cms/madie/services/QrdaService.java

```diff
@@ -37,7 +37,7 @@ public QrdaExportResponseDto generateQrda(QrdaRequestDTO request, String accessT
     QdmMeasure measure = (QdmMeasure) request.getMeasure();
     // get Libraries
     List<TranslatedLibrary> translatedLibraries =
-        translationServiceClient.getTranslatedLibraries(measure.getCql(), accessToken);
+        translationServiceClient.getTranslatedLibraries(measure.getCql(), accessToken, false);
```

Comment:
@RohitKandimalla Same question as above?

Reply:
Annotations are not used to generate QRDA, hence it doesn't really matter what is the error-severity.